packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -215,6 +216,31 @@ export const makeAnchorAsset = async (
 };
 harden(makeAnchorAsset);
 
+/** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
```

Comment:
`SwingsetVats` has `vatAdmin` as `VatAdminVat` which appears to match how it's being used here.

Reply:
I don't think `vatAdminVat` is the same as the vatAdmin device... in fact, I tested it:

```
src/proposals/startPSM.js:235:27 - error TS2345: Argument of type 'VatAdminVat' is not assignable to parameter of type 'Device<any>'.
  Type '{ createVatAdminService: (vaDevice: any) => { waitForBundleCap(bundleID: any): any; getBundleCap(bundleID: any): any; getNamedBundleCap(name: any): any; getBundleIDByName(name: any): any; createMeter(remaining: any, threshold: any): KindFacet<...>; createUnlimitedMeter(_context: any): KindFacet<...>; createVat(bundl...' is not assignable to type 'Device<any>'.
    Type '{ createVatAdminService: (vaDevice: any) => { waitForBundleCap(bundleID: any): any; getBundleCap(bundleID: any): any; getNamedBundleCap(name: any): any; getBundleIDByName(name: any): any; createMeter(remaining: any, threshold: any): KindFacet<...>; createUnlimitedMeter(_context: any): KindFacet<...>; createVat(bundl...' is not assignable to type '"Device"'.

235       const bundleCap = D(vatAdmin).getNamedBundleCap(name);
```